### PR TITLE
Update bugfix version of IoTA Lib

### PR DESCRIPTION
**Pull request by dmoranj**
Update bugfix version of IoTA Lib

### CODE REVIEW

**Comment by manucarrace** [comment]
LGTM

**Comment by jcanonav** [comment]
LGTM

**Comment by XavierVal** [comment]
LGTM

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -0,0 +1,1776 @@
...
+                      "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.0.0.tgz"
+                    }
+                  }
+                },
+                "strip-ansi": {
+                  "version": "3.0.1",
+                  "from": "strip-ansi@^3.0.0",
+                  "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
+                  "dependencies": {
+
```
Comment (fgalan) [inline npm-shrinkwrap.json]: New file in this PR? Is that correct?

**Pull request merged**